examples/scripts/sentiment_tuning.py

```diff
@@ -149,7 +150,8 @@ def collator(data):
         task_type="CAUSAL_LM",
     )
     ref_model = None
-    device_map = {"": 0}
+    # Copy the model to each device
+    device_map = {"": Accelerator().local_process_index}
```

Comment:
I noticed none of the example scripts actually work in multi-GPU settings unless we add this trick

Reply:
Makes sense!